proposals/project-smoot.md

```diff
@@ -0,0 +1,136 @@
+## Smoot: An Open Source Project for DLT Interoperability 
+
+A proposal for LFDT 
+
+# Proposor(s)
+
+- Weijia Zhang (Wanchain) [weijia@wanchain.com](mailto:weijia@wanchain.com)  
+- Other organizations (TBD)
+
+# Abstract
+
+This project proposal outlines the plan to build an open source bridge for chain-to-chain interactions, based on the Enterprise Ethereum Alliance (EEA)’s DLT Interoperability Specification, as an open source project for the Linux Foundation’s Decentralized Trust(LFDT).
+
+# Context
+
+The EEA's DLT Interoperability Specification is a standard developed by members of the Enterprise Ethereum Alliance. Several implementations have been created to connect different blockchains based on this specification, including those by Wanchain, Adhara, QualitaX, and Fnality. We aim to develop an open-source project for implementing bridges that build interoperable components connecting both EVM and non-EVM blockchains.
```

Comment:
Is the specification available publicly? If so, can we provide a link in the proposal?